Loop/View Models/BolusEntryViewModel.swift

```diff
@@ -495,7 +495,6 @@ final class BolusEntryViewModel: ObservableObject {
             isManualGlucoseEntryEnabled = false
             manualGlucoseQuantity = nil
             manualGlucoseSample = nil
-            presentAlert(.glucoseNoLongerStale)
```

Comment:
Should the enum value `glucoseNoLongerStale` be removed, and other code around it cleaned up?